cubequery/tasks/__init__.py

```diff
@@ -300,56 +298,6 @@ def standard_validation(self, args):
 
         return errors
 
-    # TODO: Bounds conversion and sometimes spatial query dependent on product
-    def validate_standard_spatial_query(self, value):

```

Comment:
Are we not validating the size of incoming polygons any more? Has that requirement been dropped?